licenses/go/bufio/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
Instead of checking the LICENSE files in, can these be gathered during container creation?

Reply:
> > I'll second the comment that we should not include the list of license files in the beam repository itself, but just add the script to the container build process to install them into the container.
> 
> I agree. It would seem better for all languages to add the license retrieval to the container build script.

I will make this change in the next commit.